stock_available_immediately/tests/test_stock_available_immediately.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Camptocamp, Akretion, Numérigraphe
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestStockLogisticsWarehouse(TransactionCase):
+
+    def test01_stock_levels(self):
+        """checking that immediately_usable_qty actually reflects \
+the variations in stock, both on product and template"""
+        moveObj = self.env['stock.move']
+        productObj = self.env['product.product']
+        templateObj = self.env['product.template']
+        supplier_location = self.env.ref('stock.stock_location_suppliers')
+        stock_location = self.env.ref('stock.stock_location_stock')
+        customer_location = self.env.ref('stock.stock_location_customers')
+        uom_unit = self.env.ref('product.product_uom_unit')
```

Comment:
uom's are managed in v12 in 'uom' module.

```suggestion
        uom_unit = self.env.ref('uom.product_uom_unit')
```